packages/core/src/routes/me.ts

```diff
@@ -0,0 +1,37 @@
+import { arbitraryObjectGuard } from '@logto/schemas';
+import { object } from 'zod';
+
+import koaGuard from '@/middleware/koa-guard';
+import { findUserById, updateUserById } from '@/queries/user';
+
+import { AuthedRouter } from './types';
+
+export default function meRoutes<T extends AuthedRouter>(router: T) {
+  router.get('/me/custom-data', async (ctx, next) => {
```

Comment:
would it cause more  frustration compare with `oidc/me`

Reply:
I was thinking about this. The interesting part of OIDC userinfo endpoint is POST is not for updating. Which makes me feel not right and decided to move "additional" or "complex" user info out of OIDC.

Reply:
Can we move these APIs to user or settings routes?

Reply:
not the same thing - double check the difference of these routes. one for admin one for current user only

Reply:
Yeah, I see. We can't have 2 different behavior in one route file. It has to be either AuthedRouter (which all requires Admin role), or AnonymousRouter

Reply:
yes

Reply:
actually 3